pg_activity

```diff
@@ -303,6 +305,8 @@ server activity monitoring.")
             # refresh the winodw
             db_info = PGAUI.data.pg_get_db_info(db_info, using_rds = options.rds)
             PGAUI.set_max_db_length(db_info['max_length'])
+            # get active connections
+            active_connections = PGAUI.data.pg_get_active_connections()    
```

Comment:
Please remove the 4 trailing spaces.

Reply:
Which editor do you use for python?
If vim can you send me the vimrc?
Thanks.

Reply:
Here it is : https://gist.github.com/julmon/674751b50a42d42db54fde99ad782f25

Reply:
Thanks. Trailing spaces removed.